lib/services/rpcbind.pm

```diff
@@ -62,8 +62,10 @@ sub check_service {
 sub check_function {
     assert_script_run("rpcinfo");
     # Wait for updated rpcinfo.
-    sleep(5);
-    assert_script_run('rpcinfo | grep nfs');
+    for (1 .. 10) {
```

Comment:
Isn't a bug there? given the fact that nfs server is up and active, we can find the port

Reply:
not a bug though, just need to wait more time for nfs be shown in the rpcinfo. thanks

Reply:
Any syslog or journal logs show up before the nfs shows up?

Reply:
I've checked the log and can't find such information, thanks

Reply:
@hjluo May you please file a bug to track this "soft failure", I don't think it is a normal state

Reply:
Thanks for the suggestion, for NFS/RPC it's not always a reliable protocol, so sometimes rpcinfo may need NFS ready to get the NFS information. I'm just not sure it's a bug.

Reply:
filed bug https://bugzilla.suse.com/show_bug.cgi?id=1193028